tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -812,15 +813,22 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
         this._lastReloadTime = new Date().toString();
       },
 
-      _fetchActiveDashboards() {
+      _computeActiveDashboard() {
+        return this._dashboardData
+            .map((d) => d.plugin)
+            .filter((dashboardName) => {
+                return this._pluginsListing[dashboardName] &&
```

Comment:
Did we want this frontend logic to be backward compatible with the
previous HTTP API?

The easiest way to make it backward compatible might be to transform the
response from `{k: v}` to `{k: typeof v === 'boolean' ? v : v.enabled}`
immediately after we receive it (in `updatePluginsListing`), allowing
most of the other logic to stay unchanged. Or you could store the actual
response data and case on it here, but it looks like as written this JS
will crash if given a response from the previous backend.